otherlibs/dynlink/dynlink_common.mli

```diff
@@ -23,6 +23,7 @@ module Make (P : Dynlink_platform_intf.S) : sig
   val is_native : bool
   val loadfile : string -> unit
   val loadfile_private : string -> unit
+  val unsafe_get_value : string -> Obj.t option
```

Comment:
Nitpicking, but shouldn't this be called `unsafe_get_global_value`?

Reply:
Fixed.  I've also added a label on the parameter.  The type of the parameter is rather unsavoury at present; I will see if I can fix that when some of the symbol type patches are presented.